lib/std/heap/sbrk_allocator.zig

```diff
@@ -58,7 +59,7 @@ pub fn SbrkAllocator(comptime sbrk: *const fn (n: usize) usize) type {
                     }
 
                     const next_addr = next_addrs[class];
-                    if (next_addr % mem.page_size == 0) {
+                    if (next_addr % heap.pageSize == 0) {
```

Comment:
Shouldn't this be `.pageSize()`? If so, how is this not being caught? Seems like a bad lack of actual test coverage.

Reply:
Yes it should. Yep it's only covered when targeting plan9 and this implementation is really bad too